app/src/test/java/com/sdp/movemeet/DistanceCalculatorTest.java

```diff
@@ -35,6 +35,7 @@ public void settingAndGettingIsCorrect() {
         final double DUMMY_LONGITUDE = 2.45;
         final double DUMMY_LATITUDE = 3.697;
         final String DUMMY_DESCRIPTION = "description";
+        final String DUMMY_DOCUMENT_PATH = "documentPath";
```

Comment:
Wouldn't it be best to use the `ActivityTest.createFakeActivity()` function once in a `@Before` setup function, and then modifying the parameters you may need to modify at the beginning of each test?

Reply:
Yes that’s a good idea! I made those modifications (apart for the case in which we have to modify the activity ID the organiser, the sport and the IDs of the participants because we don’t have any setter for those fields!). I'm however not sure I will take the risk of re-pushing my code and getting our famous Firebase error. Anyway, those changes will be available at the latest next week!